test/LOAuth.test.ts

```diff
@@ -30,7 +32,9 @@ jest.mock('../src/globals', () => ({
     },
     setInterval: jest.fn(),
     clearInterval: jest.fn(),
-    fetch: jest.fn().mockResolvedValue({ json: async () => tokenResponseJson, ok: true })
+    fetch: jest
+      .fn()
+      .mockResolvedValue({ json: async () => tokenResponseJson, ok: true })
```

Comment:
There are a lot of formatting diffs like this in this PR.  a) Did you get them all so it's consistent everywhere and b) is there a config file we could commit to enforce them going forward?  I'm cool with this PR, but I prefer that kind of thing getting a "PR of its own" for cleanliness.

Reply:
They ended up going in because of my VSCode settings formatting on save.  I'll get the formatting changes pulled out and we can add in something later if desired